account.go

```diff
@@ -120,47 +120,50 @@ func (p *PayoutScheduleParams) AppendTo(body *form.Values, keyParts []string) {
 // Account is the resource representing your Stripe account.
 // For more details see https://stripe.com/docs/api/#account.
 type Account struct {
-	ID                   string               `json:"id"`
+	BusinessLogo         string               `json:"business_logo"`
+	BusinessName         string               `json:"business_name"`
+	BusinessPrimaryColor string               `json:"business_primary_color"`
+	BusinessUrl          string               `json:"business_url"`
 	ChargesEnabled       bool                 `json:"charges_enabled"`
 	Country              string               `json:"country"`
+	DebitNegativeBal     bool                 `json:"debit_negative_balances"`
 	DefaultCurrency      string               `json:"default_currency"`
+	Deleted              bool                 `json:"deleted"`
 	DetailsSubmitted     bool                 `json:"details_submitted"`
-	PayoutsEnabled       bool                 `json:"payouts_enabled"`
-	Name                 string               `json:"display_name"`
 	Email                string               `json:"email"`
 	ExternalAccounts     *ExternalAccountList `json:"external_accounts"`
-	Statement            string               `json:"statement_descriptor"`
+	ID                   string               `json:"id"`
+	LegalEntity          *LegalEntity         `json:"legal_entity"`
+	Meta                 map[string]string    `json:"metadata"`
+	Name                 string               `json:"display_name"`
+	PayoutSchedule       *PayoutSchedule      `json:"payout_schedule"`
 	PayoutStatement      string               `json:"payout_statement_descriptor"`
-	Timezone             string               `json:"timezone"`
-	BusinessName         string               `json:"business_name"`
-	BusinessPrimaryColor string               `json:"business_primary_color"`
-	BusinessUrl          string               `json:"business_url"`
-	BusinessLogo         string               `json:"business_logo"`
-	SupportPhone         string               `json:"support_phone"`
+	PayoutsEnabled       bool                 `json:"payouts_enabled"`
+	ProductDesc          string               `json:"product_description"`
+	Statement            string               `json:"statement_descriptor"`
+	SupportAddress       *Address             `json:"support_address"`
 	SupportEmail         string               `json:"support_email"`
+	SupportPhone         string               `json:"support_phone"`
 	SupportUrl           string               `json:"support_url"`
-	ProductDesc          string               `json:"product_description"`
-	DebitNegativeBal     bool                 `json:"debit_negative_balances"`
+	Timezone             string               `json:"timezone"`
 	Type                 AccountType
-	Keys                 *struct {
-		Secret  string `json:"secret"`
+
+	Keys *struct {
```

Comment:
ooc why isn't Keys ordered alphabetically?

Reply:
Yeah, I alphabetized most things, but went back and forth on this one. These unnamed structs are a little messy so I ended up leaving them at the bottom of the block.

That said, I just changed my mind and ordered them in a5e4dd1. There aren't very many of these so I don't think it matters very much.